scripts/utils/migrate/html-to-markdown.js

```diff
@@ -145,9 +145,7 @@ turndown
   })
   .addRule('videos', {
     filter: (node) => {
-      return (
-        node.nodeName === 'IFRAME' && node.classList.contains('wistia_embed')
-      );
+      return node.nodeName === 'IFRAME';
```

Comment:
[nit] Since this is just comparing against the `nodeName`, you can use the shortcut: `filter: 'iframe'`

Reply:
ah right. make sense.